apps/backend/src/modules/projects/libs/types/types.ts

```diff
@@ -3,4 +3,6 @@ export {
 	type ProjectCreateResponseDto,
 	type ProjectGetAllItemResponseDto,
 	type ProjectGetAllResponseDto,
+	type ProjectUpdateRequestDto,
+	type ProjectUpdateResponseDto,
```

Comment:
https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/107#discussion_r1736789709
I`m not sure, but according to my task if you use patch, make sure to use 'patch' for namings as well